src/lib/bootloader/grub2efi.rb

```diff
@@ -59,7 +59,7 @@ def propose
       # for UEFI always remove PMBR flag on disk (bnc#872054)
       self.pmbr_action = :remove
 
-      grub_default.generic_set("GRUB_USE_LINUXEFI", Yast::Arch.aarch64 ? "false" : "true")
+      grub_default.generic_set("GRUB_USE_LINUXEFI", (Yast::Arch.x86_64 || Yast::Arch.i386) ? "true" : "false")
```

Comment:
well, when you touch it and also be an arm expert, can you please add a comment line above this line of code explaining why it needs to be done only on intel and not on arm? Thanks

Reply:
Ok, I added `# linuxefi/initrdefi are available on x86 only`